### PR TITLE
Implement landing page for submission portal

### PR DESCRIPTION
**Pull request by eecavanna**
In this branch, I added some new sections to the submission portal landing page.

### `TitleBanner.vue`

This component contains a page title and subtitle overlaying a banner image. The component was modeled after a header on the NMDC website.

image

> Note: The banner image is a 470 KB PNG file (i.e. `submission-portal-hero.png`). I failed to find an SVG version and don't think one exists. Given that the image contains a color gradient, I'm wondering whether a JPG version would be significantly smaller while being insignificantly lower quality.

### `IconBar.vue`

image

This component contains a row of three icon-title pairs, each pair linking to some documentation. The icons belong to the Material Design Icons library, which is used elsewhere in the code base.

### `IntroBlurb.vue`

image

This component contains three paragraphs that introduce the submission portal. In this component's name, I used "blurb" to refer to the entire set of text.

This is only visible when the user is either (a) logged out; or (b) logged in, but does not have any submissions.

### `LoginPrompt.vue`

image

This component contains a plainly-styled "alert" inviting the visitor to log into ORCiD; or create an ORCiD if they don't have one. This takes the place of the previous, loose login message.

### Components in context

Here are links to screenshots showing the components in context.

- Visitors that are not logged in will see [this](https://github.com/microbiomedata/nmdc-server/issues/1144#issuecomment-1975072897).
- Visitors that are logged in and have no submissions will see [this](https://github.com/microbiomedata/nmdc-server/issues/1144#issuecomment-1975074277).
- Visitors that are logged in and have submissions will see [this](https://github.com/microbiomedata/nmdc-server/issues/1144#issuecomment-1975073107).

---

There is one remaining task, but I don't want to hold up the review(s) for it. That remaining task is to finalize the link to the schema documentation. I will pursue that after requesting reviews as I don't expect it to have an impact on the reviews (it'll be a one-line change).

### CODE REVIEW

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                <span class="blue--text text-left text-h6 font-weight-bold">
+                  Tutorial
+                </span>
+              </v-col>
+            </v-row>
+          </v-container>
+        </a>
+      </v-col>
+      <v-col>
+        <!-- TODO: Verify that this is the correct `href`, given recent decision about documentation duplication. -->
```
Comment (eecavanna) [inline web/src/views/SubmissionPortal/Components/IconBar.vue:68]: TODO: @eecavanna

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <a
+          target="_blank"
+          title="View the metadata schema"
+          href="https://nmdc-documentation.readthedocs.io/en/latest/reference/metadata/combined_schema_docs.html"
+          class="text-decoration-none"
+        >
```
Comment (eecavanna) [inline web/src/views/SubmissionPortal/Components/IconBar.vue:74]: TODO: @eecavanna

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import {
+  defineComponent,
+} from '@vue/composition-api';
+
+export default defineComponent({
+  components: {},
+  setup() {
+    return {};
+  },
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/IntroBlurb.vue:10]: remove

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <v-row>
+      <v-col class="px-0">
+        <h2 class="blue--text text-h6">
+          Supporting FAIR data
+        </h2>
+        <p class="text-justify mb-0 text-body-1">
+          The Submission Portal leverages validation functions of the
+          <a
+            href="https://github.com/cidgoh/DataHarmonizer"
+            target="_blank"
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/IntroBlurb.vue:35]: add `rel = noreferrer nopener` for extra security.

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            title="View cidgoh/DataHarmonizer on GitHub"
+          >DataHarmonizer</a>
+          tool to check entered metadata values against the standards in the
+          <a
+            href="https://github.com/microbiomedata/nmdc-schema"
+            target="_blank"
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/IntroBlurb.vue:41]: add `rel = noreferrer nopener` for extra security.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 6]
```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  props: {},
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/LoginPrompt.vue:5]: remove

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+          </p>
+          <p class="body-2 mb-0 text--disabled">
+            Don't have an ORCiD? Individuals can
+            <a
+              href="https://support.orcid.org/hc/en-us/articles/360006973953"
+              target="_blank"
+              title="Learn about registering an ORCiD"
+              class="text--disabled"
+            >get one for free</a>.
+          </p>
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/LoginPrompt.vue:34]: I am concerned about the readability of this text. The contrast from the background is low.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        <v-col class="justify-content-center">
+          <p class="body-1">
+            Login with your ORCiD
+            to access the Submission Portal.
+          </p>
+          <p class="body-2 mb-0 text--disabled">
+            Don't have an ORCiD? Individuals can
+            <a
+              href="https://support.orcid.org/hc/en-us/articles/360006973953"
+              target="_blank"
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/LoginPrompt.vue:30]: add rel = noreferrer nopener for extra security.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+              title="Learn about registering an ORCiD"
+              class="text--disabled"
+            >get one for free</a>.
+          </p>
+        </v-col>
+        <v-col class="flex-grow-0">
+          <!-- Note: This button is a simplified version of: `web/src/components/Presentation/AuthButton.vue` -->
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/LoginPrompt.vue:37]: I don't think we'll need this comment long term, can probably remove it.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+      loginHref: '/login',
+    };
+  },
+});
+</script>
+
+<template>
+  <v-alert
+    class="mb-0"
+    text="text"
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/LoginPrompt.vue:17]: text is a boolean prop so no need to set a value

File: web/src/views/SubmissionPortal/Components/TitleBanner.vue [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+  components: {},
+  setup() {
+    return {};
+  },
+});
+</script>
+
+<template>
+  <v-container>
+    <v-row :style="{ backgroundImage: 'url(' + require('@/assets/submission-portal-hero.png') + ')', backgroundSize: 'cover' }">
```
Comment (marySalvi) [inline web/src/views/SubmissionPortal/Components/TitleBanner.vue:16]: Hooray! Setting an image to background image when it is not info bearing is excellent accessibility practice. :)

**Review by marySalvi — CHANGES_REQUESTED** [review]
Some minor changes but overall looks great!

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 12]
```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+import {
+  defineComponent,
+} from '@vue/composition-api';
+
+export default defineComponent({
+  components: {},
+  setup() {
+    return {};
+  },
```
[the remark this answers is not in the document]
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IconBar.vue]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue (hunk 3 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IntroBlurb.vue]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue (hunk 8 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue:30]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue (hunk 9 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue (hunk 10 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue]: Thanks for bringing that to my attention! Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+                </span>
+              </v-col>
+            </v-row>
+          </v-container>
+        </a>
+      </v-col>
+      <v-col>
+        <!-- TODO: Verify that this is the correct `href`, given recent decision about documentation duplication. -->
+        <a
+          target="_blank"
```
[the remark this answers is not in the document]
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IconBar.vue:67]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+                  Quickstart
+                </span>
+              </v-col>
+            </v-row>
+          </v-container>
+        </a>
+      </v-col>
+      <v-col>
+        <a
+          target="_blank"
```
[the remark this answers is not in the document]
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IconBar.vue:40]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IconBar.vue [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+  },
+});
+</script>
+
+<template>
+  <v-container class="mt-4 limited-width">
+    <v-row>
+      <v-col>
+        <a
+          target="_blank"
```
[the remark this answers is not in the document]
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IconBar.vue:14]: Thank you for providing the "fix" and a link to some additional information! Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue (hunk 7 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue]: Thank you for bringing this to my attention, @marySalvi!

I plugged in the colors into [an online tool](https://webaim.org/resources/contrastchecker/) and got "failing grades":

image

I will increase the contrast (TBD).

Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue]: I changed it to black in commit 6865d7c86fe6bb33a5c764a6645b0537b32be8a0. That online tool shows "passing" all across the board, for black (on gray).

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue (hunk 6 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue (hunk 5 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IntroBlurb.vue:38]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

File: web/src/views/SubmissionPortal/Components/IntroBlurb.vue (hunk 4 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IntroBlurb.vue:31]: Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.

**Comment by eecavanna** [comment]
I have addressed all the review feedback.

FYI: I am still waiting on guidance from a team member regarding where one of the links will point. I'll reach out to her again about it.

File: web/src/views/SubmissionPortal/Components/IconBar.vue (hunk 2 above)
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/IconBar.vue]: I'll update it to point to: https://w3id.org/nmdc/about

**Comment by eecavanna** [comment]
I'm ready for this to be merged in (assuming nobody objects to its contents).

**Review by marySalvi — APPROVED** [review]

**Review by naglepuff — CHANGES_REQUESTED** [review]
Sorry to swoop in last minute like this but the "i" in ORCID should be capitalized. The "i" in ORCID iD is lowercase.

File: web/src/views/SubmissionPortal/Components/LoginPrompt.vue [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+<template>
+  <v-alert
+    class="mb-0"
+    text
+  >
+    <v-container>
+      <v-row align="center">
+        <v-col class="justify-content-center">
+          <p class="body-1">
+            Login with your ORCiD
```
[the remark this answers is not in the document]
Comment (eecavanna) [reply web/src/views/SubmissionPortal/Components/LoginPrompt.vue:22]: I'll make this change now.

**Pull request merged**